src/source_diff.cc

```diff
@@ -54,6 +75,11 @@ Source::DiffView::DiffView(const boost::filesystem::path &file_path) : Gsv::View
   renderer->tag_removed_below=get_buffer()->create_tag();
   renderer->tag_removed_above=get_buffer()->create_tag();
   
+  add_events(Gdk::BUTTON_PRESS_MASK | Gdk::BUTTON_RELEASE_MASK);
```

Comment:
I don't know for what events you're already checking, so I added this to be sure, but it's probably redundant